package.json

```diff
@@ -16,7 +16,6 @@
     "build": "npm install && node script/barrelize.js && tsc && npm run copy",
     "prebuild:release": "script/prebuild",
     "build:release": "npm run build && npm run lint && npm run check && npm run doc && npm run test:retry",
-    "postbuild:release": "script/postbuild",
```

Comment:
The postbuild script is useful to run locally as it tells you whether your build is likely to pass CI. What is the motivation behind removing it?